synapse/axon.py

```diff
@@ -451,6 +453,9 @@ def __init__(self, axondir, **opts):
 
         corepath = os.path.join(self.axondir, 'axon.db')
         self.core = s_cortex.openurl('sqlite:///%s' % corepath)
+        for name, model in axon_model.getBaseModels():
+            self.core.addDataModel(name, model)
```

Comment:
I tried that, but get `NoSuchOpt`. Do I really need to define config options for my data model? What's wrong with calling `addDataModel`?